NEWS.md

```diff
@@ -1,3 +1,72 @@
+# Sparklyr 1.3.0
+
+### Spark ML
+
+- `ml_evaluate()` methods are implemented for ML clustering and classification models
+
+### Distributed R
+
+- Created helper methods to integrate Spark SQL higher-order functions with
+  `dplyr::mutate`
+
```

Comment:
I think this one fits better under `Data`, so removing form this section and suggesting another edit bellow:

```suggestion
- Created helper methods to integrate Spark SQL higher-order functions with
  `dplyr::mutate`

```